migrations/20170906161500-healthcheck_interval.js

```diff
@@ -0,0 +1,53 @@
+module.exports = {
+    up: function (queryInterface, DataTypes) {
+        let changes = [];
+
+        // healthcheck_interval
+        changes.push(
+            queryInterface.addColumn(
+                'Ports',
+                'healthcheck_interval',
+                {
+                    type: DataTypes.INTEGER,
+                    field: "healthcheck_interval",
+                    validate: {
+                        min: 1,
+                        max: 3600
+                    },
+                    defaultValue: 10,
```

Comment:
don't we want the default to be 1, since that is what most people have today. Then go back and switch the default to 10, in a separate release?

Reply:
I think you are confusing healthcheck_timeout with interval.  Today interval is a 10 second default because we don't overwrite what kubernetes has set.  Just matching that with a value we can change in the future.

Reply:
It's a new field, no one has it set yet, right?

Reply:
ok yes you are right. Thought interval was 1 second also. 

Carry on.